libraries/libwhb/src/gfx.c

```diff
@@ -310,14 +314,22 @@ WHBGfxInit()
    sDrcRenderMode = GX2GetSystemDRCMode();
    sTvSurfaceFormat = GX2_SURFACE_FORMAT_UNORM_R8_G8_B8_A8;
    sDrcSurfaceFormat = GX2_SURFACE_FORMAT_UNORM_R8_G8_B8_A8;
-
+   
+   AVMSetTVAspectRatio(sTvAspectRatio);
```

Comment:
I assume you meant to use `AVMGetTVAspectRatio` here instead. The proper way to do this would be using `GX2GetSystemTVAspectRatio` (see #384), to avoid depending on the lower level AVM library entirely.

Reply:
#384 has been merged.